Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Frank Hellmann <hellmann@pik-potsdam.de>, Michael Lindner <michaelli
 version = "0.5.4"
 
 [deps]
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
```

Comment:
Would it be possible to add this dependency to a local environment in the `benchmarking` folder instead?

Reply:
This is not that trivial unfortunately. Normally, `pkgbenchmark` will take the benchmark files from the specific commit you are benchmarking. I think this is not what we want here. During the process of benchmarking a different commit the script will
- copy the NetworkDynamics repo to a temp location
- copy the benchmark folder to a different temp location
- change the temp NetworkDynamics folder to the commit to we want to
- at this point, it is not clear how to reach the temp ND folder from the environment which lives inside the benchmark folder, therefore it is much simpler to run everything from the ND environment